css/about.css

```diff
@@ -0,0 +1,178 @@
+@import url("https://fonts.googleapis.com/css2?family=Paytone+One&family=Rubik+Mono+One&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital@1&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Alfa+Slab+One&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.text2 {
+  background-color: gray;
+  text-align: center;
+  padding: 15px 5px;
+  font-family: "Lato", sans-serif;
+}
+
+.contact {
+  text-align: center;
+  color: orangered;
+  font-size: 16px;
+  margin-bottom: 0;
+}
+
+.contact span {
+  font-weight: bold;
+  font-family: "Lato", sans-serif;
+}
+
+.creative {
+  font-size: 26px;
+  text-align: center;
+  font-family: "Lato", sans-serif;
+  margin-bottom: 20px;
+}
+
+.line {
+  width: 200px;
+}
+
+.line2 {
+  margin-bottom: 20px;
+}
+
+.log,
+.past-info {
+  text-align: center;
+  font-family: "Lato", sans-serif;
+  font-size: 17px;
+  font-weight: 400;
+  color: darkslategray;
+}
+
+.logo-img2 {
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```